crud/compare/filters.lua

```diff
@@ -264,25 +270,62 @@ local function add_collation_postfix(func_name, value_opts)
     error('Unsupported collation: ' .. tostring(value_opts.collation))
 end
 
+local function cond_string(cond, values_opts, field, j)
+    local value = cond.values[j]
+    local value_type = cond.types[j]
+    local value_opts = values_opts[j] or {}
+
+    local func_name = 'eq'
+
+    if value_type == 'string' then
+        func_name = add_collation_postfix('eq', value_opts)
+        if collations.is_unicode(value_opts.collation) then
+            func_name = add_strict_postfix(func_name, value_opts)
+        end
+    elseif value_type == 'uuid' then
+        func_name = 'eq_uuid'
+    end
+
+    return format_comp_with_value(field, func_name, value)
+end
+
 local function format_eq(cond)
     local cond_strings = {}
     local values_opts = cond.values_opts or {}
 
+    if #cond.values == 1 then
```

Comment:
Multivalue equity select and range operators are different features, so it's better to implement them in separate commits (or maybe even separate PRs).